src/types/types.ts

```diff
@@ -0,0 +1,125 @@
+export interface YouTubeApi {
```

Comment:
it's better to split interfaces in different files, so you could import only needed parts

Reply:
Could be omitted due to project needs and kept in mind about separation in future